File: Android/Sample/app.ok-if-stub/src/main/java/net/wequick/example/small/appok_if_stub/MyReceiver.java
package net.wequick.example.small.appok_if_stub;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.widget.Toast;

/**
 * Created by galen on 2016/11/4.
 */
public class MyReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        Toast.makeText(context, "Hi there, I'm MyReceiver.", Toast.LENGTH_SHORT).show();
    }
}

File: Android/Sample/app.ok-if-stub/src/main/java/net/wequick/example/small/appok_if_stub/MyReceiver.kt
package net.wequick.example.small.appok_if_stub

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.widget.Toast

/**
 * Created by galen on 2016/11/4.
 */
class MyReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        Toast.makeText(context, "Hi there, I'm MyReceiver.", Toast.LENGTH_SHORT).show()
    }
}
